strawberry_django/mutations/fields.py

```diff
@@ -225,6 +227,32 @@ def arguments(self, value: list[StrawberryArgument]):
         args_prop = super(DjangoMutationBase, self.__class__).arguments
         return args_prop.fset(self, value)  # type: ignore
 
+    def refetch(self, resolved: _T, *, info: Info | None) -> _T:
+        if not DjangoOptimizerExtension.enabled or info is None:
```

Comment:
`DjangoOptimizerExtension.enabled` is a `ContextVar`, I think `.get()` is missing here.